### PR TITLE
Add customization options for Bulls and Cows with PT translation

### PR DESCRIPTION
**Pull request by leocaseiro**
#### Checklist

* [x] Use a separate branch in your local repo (not `master`).
* [x] I've read and agree with the [contribution guidelines](https://www.freeboardgames.org/docs/?path=/story/documentation-contributing--page).

Also, I made it as AI, because the game is single-player. As a workaround, I just move from AI to the player, and the game keeps working as a single-player.

The customization is to decrease/increase difficulty.

### CODE REVIEW

File: web/src/games/bullsAndCows/locales/pt.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Touros & Vacas",
+  "description": "parecido com Senha",
+  "descriptionTag": "Jogue Touros e Vacas (também conhecido como Senha) de graça online.",
+  "instructions": {
+    "videoId": "GG7acm844UE",
```
Comment (leocaseiro) [inline web/src/games/bullsAndCows/locales/pt.json:6]: On my tests it seems that the video still showing the original (english one), did I miss something?

**Comment by vdfdev** [comment]
This looks great! LGTM, we just need to figure out why it broke CI

File: web/src/games/bullsAndCows/locales/pt.json (hunk 1 above)
Comment (vdfdev) [reply web/src/games/bullsAndCows/locales/pt.json:6]: I am not sure, I can try to debug this weekend

**Comment by leocaseiro** [comment]
Fixed the tests. I shouldn't have trusted on my changes, but I played the game fine locally.

Anyway, CI is 🟢 now!

**Comment by leocaseiro** [comment]
Just updated the branch!

File: web/src/games/bullsAndCows/locales/pt.json (hunk 1 above)
Comment (vdfdev) [reply web/src/games/bullsAndCows/locales/pt.json:6]: So, after we got a cease and desist from another game, we don't want to have any reference in videos to other (copyrighted) games. The only exception is "similar to X" which is protected by copyright law.

This video has references to the other game, so can you please remove the video?

File: web/src/games/bullsAndCows/locales/en.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Bulls & Cows",
+  "description": "similar to Mastermind",
+  "descriptionTag": "Play Bulls and Cows (also known as Mastermind) for free online.",
+  "instructions": {
+    "videoId": "dMHxyulGrEk",
```
Comment (vdfdev) [inline web/src/games/bullsAndCows/locales/en.json:6]: Here too, please remove the video

File: web/src/games/bullsAndCows/locales/en.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Bulls & Cows",
+  "description": "similar to Mastermind",
+  "descriptionTag": "Play Bulls and Cows (also known as Mastermind) for free online.",
```
Comment (vdfdev) [inline web/src/games/bullsAndCows/locales/en.json:4]: The wording should be "similar to Mastermind", not also known as.

**Review by vdfdev — APPROVED** [review]

**Comment by vdfdev** [comment]
FYI, just need to fix some little issues regarding copyright before merging. I can fix them for you this weekend too

**Comment by leocaseiro** [comment]
Good point! Done it!

I found an English and Portugues video for the non-streamed version of the games

File: web/src/games/bullsAndCows/locales/pt.json (hunk 1 above)
Comment (leocaseiro) [reply web/src/games/bullsAndCows/locales/pt.json:6]: It's working fine EN/PT :D

File: web/src/games/bullsAndCows/locales/en.json (hunk 2 above)
Comment (leocaseiro) [reply web/src/games/bullsAndCows/locales/en.json:6]: Done

File: web/src/games/bullsAndCows/locales/en.json (hunk 3 above)
Comment (leocaseiro) [reply web/src/games/bullsAndCows/locales/en.json:4]: Done

**Comment by vdfdev** [comment]
The point is not about being streamed or not... It mentions "mastermind" which can lead to copyright issues... A video to be acceptable here is either from a game in the public domain (like chess) or a video made specifically for our game (using bulls and cows name)

**Comment by leocaseiro** [comment]
Sorry, that's what I meant about (streamed), I should've used the term "copyrighted games".

Yeah, I made sure I selected games that are not copyrighted for both videos.

**Comment by vdfdev** [comment]
Ah, I see! Great!

**Comment by vdfdev** [comment]
I'm going to push to prod tomorrow

**Pull request merged**